cgo.go

```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2022 The Ebitengine Authors
+
+//go:build cgo
+// +build cgo
+
+package purego
+
+// if CGO_ENABLED=1 import the Cgo runtime to ensure that it is set up properly.
+// This is required since some frameworks need TLS setup the C way which Go doesn't do.
+// We currently don't support ios in fakecgo mode so force Cgo or fail
+import _ "runtime/cgo"
```

Comment:
Do we really need this file? What if we don't have this file but a user specifies cgo? In this case, `runtime/cgo` is automatically imported, right?

Reply:
We do need it. Even if `CGO_ENABLED=1` if there are no usages of `import "C"` or `import "runtime/cgo` then the Cgo runtime is NOT compiled in. Normally, it would not be possible to call into C without using one of those imports but with purego it is.

Reply:
I see, that's a good point.